README.md

```diff
@@ -0,0 +1,50 @@
+# Scverse Ecosystem Packages
+
+This repository contains the list of scverse ecosystem packages that are displayed on scverse.org.
+The goal is to increase visibility of ecosystem packages and make it easier for users to find appropriate software.
+In the future, we may also test releases of core packages against the test suites of ecosystem packages.
+
+If a package is part of this list, it means it fulfills certain minimum requirements as outlined below.
+It **does not** mean we endorse the package or performed an in-depth review.
+
+**Hint:** If you want to receive notifications about new ecosystem packages, simply use GitHub's "watch" functionality for this repository.
+
+## How can my package become part of the list?
+
+Submit a pull-request adding a `meta.yaml` file for your package to the `packages` directory.
+
+-   Please refer to other entries for examples
+-   The full definition of available fields is available in [`schema.json`](schema.json)
+-   Please copy the checklist from below into the pull request description and answer all questions.
+
+## What are the requirements for an ecosystem package?
+
+For a package to become an approved ecosystem package, it must fulfill all mandatory requirements from the checklist below.
+
+Ecosystem packages can be written in non-Python languages as long as they fulfill the above requirements.
+
+If you cannot or do not want to comply with these requirements, you are still free to make your package interoperable with scverse by using our datastructures, but we will not list your package on our ecosystem page.
+
+## Checklist for adding packages
+
+### Mandatory
```

Comment:
Why is this section separate from the requirements one above?

Also, I don't think it needs to be a checklist unless it's in the PR template.

Reply:
The details are gone now from the above section... any chance you've been looking at an old version? 
But we can also make it a subheading...

As I mentioned in the other PR, I didn't like the PR template too much - I think the full checklist should be more accessible than being hidden in a PR template. 

We can wrap this up tonight.